Opdracht1.js

```diff
@@ -0,0 +1,47 @@
+//Opdracht 1 - Array Methoden
+
+//opdracht 1a :Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+
+
+let types = [];
+
+for (let i = 0; i < inventory.length; i++) {
+    types.push(inventory[i].type)
+}
+
+console.log(types)
+
+//Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+
+const soldOut = inventory.filter((inv) =>{
+    return inv.originalStock - inv.sold === 0;
+});
```

Comment:
Dit zou je nog wat korter kunnen schrijven
```suggestion
const soldOut = inventory.filter(inv => inv.originalStock === inv.sold);
```